File: app/src/main/java/com/forkmang/activity/TutorialScreen.java
package com.forkmang.activity;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.forkmang.R;

public class TutorialScreen extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_tutorial_screen);
        TextView txtSkip = findViewById(R.id.txtSkip);
        txtSkip.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                final Intent mainIntent = new Intent(TutorialScreen.this, SelectLanguage.class);
                startActivity(mainIntent);
                finish();
            }
        });

    }
}

File: app/src/main/java/com/forkmang/activity/TutorialScreen.kt
package com.forkmang.activity

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.forkmang.R


class TutorialScreen : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_tutorial_screen)
        val txtSkip: TextView = findViewById(R.id.txtSkip)
        txtSkip.setOnClickListener {
            val mainIntent: Intent = Intent(this@TutorialScreen, SelectLanguage::class.java)
            startActivity(mainIntent)
            finish()
        }
    }
}